### PR TITLE
Enable code coverage during pull request builds

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -36,3 +36,29 @@ build_script:
...
     Get-ChildItem Setup\GitExtensions-Portable-*.zip | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name }
+
+test_script:
+- ps: |
+    $testAssemblies = @(
+        "UnitTests\GitCommandsTests\bin\Release\GitCommandsTests.dll"
+        "UnitTests\GitUITests\bin\Release\GitUITests.dll"
+        "UnitTests\GravatarTests\bin\Release\GravatarTests.dll"
+        "UnitTests\Plugins\ReleaseNotesGeneratorTests\bin\Release\ReleaseNotesGeneratorTests.dll"
+        "UnitTests\ResourceManagerTests\bin\Release\ResourceManagerTests.dll"
```
Comment (RussKie) [inline appveyor.yml]: Can we do a wildcard, something like `UnitTests\**\bin\Release\*Tests.dll`?

Comment (sharwell) [reply appveyor.yml]: I imagine it's possible. For this pull request, I just started with the list of test assemblies reported by one of the existing AppVeyor master branch builds to define this array. Would you be comfortable with leaving the change as a follow-up item?

Comment (RussKie) [reply appveyor.yml]: Sure, nw

**Comment by RussKie** [comment]
Thank you

**Pull request merged**